app/schemas/users.js

```diff
@@ -39,3 +39,28 @@ exports.userSchema = {
     errorMessage: 'Password only should have alphanumeric characters'
   }
 };
+
+exports.signInSchema = {
+  email: {
+    in: ['body'],
+    isEmpty: {
+      negated: true,
+      errorMessage: 'email is required'
+    },
+    custom: {
+      options: email => emailRegex.test(email),
+      errorMessage: 'email not part of Wolox'
+    },
+    errorMessage: 'something is wrong with the email'
+  },
+  password: {
+    isEmpty: {
+      negated: true,
+      errorMessage: 'password is required'
+    },
+    isLength: {
+      errorMessage: 'Password should be at least 8 characters',
+      options: { min: 8 }
```

Comment:
I would think that this validation is not necessary, also it would not be good to give indications of the length of the password.